versions.tf

```
@@ -1,7 +1,21 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
 terraform {
-  required_version = ">= 1.1.7"
+  required_version = ">= 1.5.5"
```

```suggestion
  required_version = "<= 1.5.5"
```

Version should be less than or equal to 1.5.5 (not greater than), this is the last version before Terraform's license changed.

---

done